staff/NuriaComa/cv/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Currículum Vitae</title>
+</head>
+<body>
+    <h1> CV_Núria Coma</h1>
```

Comment:
"CV_name" seems like a file name, instead of an actual in-file heading. I recommend changing to just your name maybe?

Reply:
Done!